lib/jsonfile.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+
+from . import errors
```

Comment:
Because `errors` only could mean another installed package via a dependency. Relative or absolute imports of modules of the same package should be preferred. Given that mozmill-ci is not a real package, I have to use the relative path.